gluten-ut/common/src/test/scala/io/glutenproject/utils/BackendTestSettings.scala

```diff
@@ -92,6 +94,14 @@ abstract class BackendTestSettings {
       exclusion.add(Exclude(testNames: _*))
       this
     }
+    def includeGlutenTest(testName: String*): SuiteSettings = {
+      inclusion.add(IncludeGlutenTest(testName: _*))
+      this
+    }
+    def excludeGlutenTest(testName: String*): SuiteSettings = {
+      exclusion.add(ExcludeGlutenTest(testName: _*))
+      this
+    }
     def includeByPrefix(prefixes: String*): SuiteSettings = {
```

Comment:
Do we also want to includeGlutenTestByPrefix and excludeGlutenTestByPrefix potentially in the future?

Reply:
yeah, it will be made up later